client/src/ui/track/JobCard.tsx

```diff
@@ -115,9 +145,72 @@ export function JobCard({
             customSx={{
               height: "20px",
               fontSize: "10px",
-              width: "70px", // Adjust width for fit
+              width: "70px",
             }}
           />
+          {/* Display the score if status is OA */}
+          {Status === "Online Assesstment" && currentScore && outOfScore && (
+            <Typography
+              variant="caption"
+              sx={{
+                backgroundColor: "#496FFF",
+                color: "white",
+                borderRadius: "8px",
+                padding: "2px 6px",
+                fontSize: "10px",
+              }}
+            >
+              {currentScore}/{outOfScore}
+            </Typography>
+          )}
+          {/* Display the score if status is OA */}
+          {Status === "Interviewing" && (
+            <Select
+              value={interviewingRound || ""}
+              onChange={handleInterviewRoundChange}
+              renderValue={(selected) => `R: ${selected}`}
+              sx={{
+                height: "20px",
+                fontSize: "10px",
+                backgroundColor: "#496FFF",
+                color: "white",
+                borderRadius: "8px",
+                width: "auto",
+                "& .MuiSelect-icon": {
+                  color: "white",
+                },
+                "&.MuiOutlinedInput-root": {
+                  "& fieldset": {
+                    border: "none",
+                  },
+                },
+              }}
+              MenuProps={{
+                PaperProps: {
+                  sx: {
+                    borderRadius: "8px",
+                    boxShadow: "none",
+                    backgroundColor: "#496FFF",
+                  },
+                },
+              }}
+            >
+              {["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"].map(
```

Comment:
we can do a max of 4 rounds. we can just render 4+ if the interview exceeds 4 rounds

Reply:
Yes it has been fixed also